lib/langchainrb_rails/generators/langchainrb_rails/templates/create_prompts.rb.tt

```diff
@@ -0,0 +1,15 @@
+class CreatePrompts < ActiveRecord::Migration[6.0]
```

Comment:
You should be able to use something like this:
```ruby
class CreatePrompts < ActiveRecord::Migration<%= migration_version %>
```
to avoid hardcoding `6.0`.